pkg/api/unversioned.go

```diff
@@ -49,3 +53,12 @@ func FieldSelectorQueryParam(version string) string {
 	}
 	return "field-selector"
 }
+
+// String returns available api versions as a human-friendly version string.
+func (apiVersions APIVersions) String() string {
+	return strings.Join(apiVersions.Versions, ",")
+}
+
+func (apiVersions APIVersions) GoString() string {
```

Comment:
This seems redundant with the above. Also I'm not seeing where either of these are used...